plugins/quay/src/components/QuayRepository/tableHeading.tsx

```diff
@@ -6,7 +6,7 @@ import { Tooltip } from '@material-ui/core';
 import makeStyles from '@material-ui/core/styles/makeStyles';
 
 import { vulnerabilitySummary } from '../../lib/utils';
-import type { Layer } from '../../types';
+import type { QuayTagData } from '../../types';
 
 export const columns: TableColumn[] = [
```

Comment:
```suggestion
export const columns: TableColumn<QuayTagData>[] = [
```
With this change, you wouldn't need to typecast `quayRowData`